### PR TITLE
feat(behavior_velocity): add external velocity limit to behavior_velocity

### PR DESCRIPTION
**Pull request by taikitanaka3**
Signed-off-by: tanaka3 <ttatcoder@outlook.jp>

## PR Type

- New Feature

## Related Links

https://github.com/autowarefoundation/autoware.universe/pull/497

## Description

add external velocity limit to consider smoothed velocity in behavior velocity planner
## Review Procedure

note: currently behavior velocity only apply Analytical smoother type and other method is not yet tested

## Remarks

maybe conflict with this PR https://github.com/tier4/autoware_launch/pull/271

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py [hunk 1]
```diff
@@ -178,6 +178,39 @@ def generate_launch_description():
         extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
     )
 
+    # smoother param
+    common_param_path = os.path.join(
+        get_package_share_directory("planning_launch"),
+        "config",
+        "scenario_planning",
+        "common",
+        "Analytical.param.yaml",
```
Comment (mkuri) [inline planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py]: common.param.yaml?

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py [hunk 2]
```diff
@@ -178,6 +178,39 @@ def generate_launch_description():
...
+    with open(base_param_path, "r") as f:
+        base_param = yaml.safe_load(f)["/**"]["ros__parameters"]
+
+    smoother_param_path = os.path.join(
+        get_package_share_directory("planning_launch"),
+        "config",
+        "scenario_planning",
+        "common",
+        "motion_velocity_smoother",
+        "Analytical.param.yaml",
```
Comment (mkuri) [inline planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py:209]: I think smoother's altorithm type should be selectable.
https://github.com/tier4/autoware_launch/blob/tier4/universe/planning_launch/launch/scenario_planning/scenario_planning.launch.xml#L26-L33

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py (hunk 1 above)
Comment (taikitanaka3) [reply planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py]: yes right I missed sync with experiment branch

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py (hunk 2 above)
Comment (taikitanaka3) [reply planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py:209]: @mkuri 
however smoother type is already define in behavior velocity so currently Analytical is only supported

Comment (taikitanaka3) [reply planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py:209]: @mkuri 
currently behavior velocity only apply Analytical smoother type and other method is not yet tested

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py (hunk 1 above)
Comment (taikitanaka3) [reply planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py]: [68a621f](https://github.com/tier4/autoware_launch/pull/238/commits/68a621ff916e6d004ac77ab433bfdda1bdd39c28)

**Review by mkuri — APPROVED** [review]
I have confirmed that the behavior velocity planner node works on the planning simulator

**Pull request merged**